projects/cert-manager/cert-manager/helm/patches/0001-Use-sourceRegistry-and-digest-in-chart.patch

```diff
@@ -383,74 +375,91 @@ index 35ec9766a..fd30fc787 100644
  
  # Override the namespace used to store DNS provider credentials etc. for ClusterIssuer
-@@ -333,7 +336,7 @@ webhook:
-   serviceLabels: {}
+@@ -753,7 +756,7 @@ webhook:
  
-   image:
+     # The container image for the cert-manager webhook
+     # +docs:property
 -    repository: quay.io/jetstack/cert-manager-webhook
 +    repository: cert-manager/cert-manager-webhook
-     # You can manage a registry with
-     # registry: quay.io
-     # repository: jetstack/cert-manager-webhook
-@@ -344,6 +347,7 @@ webhook:
  
-     # Setting a digest will override any tag
+     # Override the image tag to deploy by setting this variable.
+     # If no value is set, the chart's appVersion will be used.
+@@ -765,6 +768,7 @@ webhook:
      # digest: sha256:0e072dddd1f7f8fc8909a2ca6f65e76c5f0d2fcfb8be47935ae3457e8bbceb20
-+    digest: {{cert-manager/cert-manager-webhook}}
  
+     # Kubernetes imagePullPolicy on Deployment.
++    digest: {{cert-manager/cert-manager-webhook}}
      pullPolicy: IfNotPresent
  
-@@ -471,7 +475,7 @@ cainjector:
-   podLabels: {}
+   serviceAccount:
+@@ -1036,7 +1040,7 @@ cainjector:
  
-   image:
+     # The container image for the cert-manager cainjector
+     # +docs:property
 -    repository: quay.io/jetstack/cert-manager-cainjector
 +    repository: cert-manager/cert-manager-cainjector
-     # You can manage a registry with
-     # registry: quay.io
-     # repository: jetstack/cert-manager-cainjector
-@@ -482,6 +486,7 @@ cainjector:
  
-     # Setting a digest will override any tag
+     # Override the image tag to deploy by setting this variable.
+     # If no value is set, the chart's appVersion will be used.
+@@ -1048,6 +1052,7 @@ cainjector:
      # digest: sha256:0e072dddd1f7f8fc8909a2ca6f65e76c5f0d2fcfb8be47935ae3457e8bbceb20
+ 
+     # Kubernetes imagePullPolicy on Deployment.
 +    digest: {{cert-manager/cert-manager-cainjector}}
+     pullPolicy: IfNotPresent
+ 
+   serviceAccount:
+@@ -1095,7 +1100,7 @@ acmesolver:
  
+     # The container image for the cert-manager acmesolver
+     # +docs:property
+-    repository: quay.io/jetstack/cert-manager-acmesolver
++    repository: cert-manager/cert-manager-acmesolver
+ 
+     # Override the image tag to deploy by setting this variable.
+     # If no value is set, the chart's appVersion will be used.
+@@ -1107,6 +1112,7 @@ acmesolver:
+     # digest: sha256:0e072dddd1f7f8fc8909a2ca6f65e76c5f0d2fcfb8be47935ae3457e8bbceb20
+ 
+     # Kubernetes imagePullPolicy on Deployment.
++    digest: {{cert-manager/cert-manager-acmesolver}}
      pullPolicy: IfNotPresent
  
-@@ -577,7 +582,7 @@ startupapicheck:
-   podLabels: {}
+ # +docs:section=Startup API Check
+@@ -1222,7 +1228,7 @@ startupapicheck:
  
-   image:
--    repository: quay.io/jetstack/cert-manager-ctl
+     # The container image for the cert-manager startupapicheck
+     # +docs:property
+-    repository: quay.io/jetstack/cert-manager-startupapicheck
 +    repository: cert-manager/cert-manager-ctl
```

Comment:
Why are we using `cert-manager-ctl` repository for `cert-manager-startupapicheck`?

From the [v1.14.0](https://github.com/cert-manager/cert-manager/releases/tag/v1.14.0) changelog,
```
🔧 Breaking Changes
The startupapicheck job uses a new OCI image called "startupapicheck", instead of the ctl image.
If you run in an environment in which images cannot be pulled, be sure to include the new image.
```